assets/dev/js/editor/document/elements/commands/browser-import.js

```diff
@@ -0,0 +1,22 @@
+import CommandBase from 'elementor-api/modules/command-base';
+import BrowserImportManager from '../../../utils/browser-import/manager';
+
+export class BrowserImport extends CommandBase {
+	validateArgs( args ) {
+		this.requireContainer( args );
```

Comment:
Maybe validate `items`

Reply:
> Maybe validate `items`

`items` can be almost anything - from a base64 string to an array of `Blob`s.